app/com/criteo/drelephant/heuristics/GarmadonTransferHeuristic.java

```diff
@@ -0,0 +1,117 @@
+package com.criteo.drelephant.heuristics;
+
+import com.avaje.ebean.Ebean;
+import com.avaje.ebean.SqlRow;
+import com.avaje.ebean.Transaction;
+import com.linkedin.drelephant.analysis.Severity;
+import com.linkedin.drelephant.util.Utils;
+import models.AppHeuristicResult;
+import models.AppHeuristicResultDetails;
+import models.AppResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class GarmadonTransferHeuristic {
+    private static final Logger LOGGER = LoggerFactory.getLogger(GarmadonTransferHeuristic.class);
+
+    private static final String TABLENAME_PREFIX = "garmadon_";
+    private static final String HEURISTIC_RESULT_TABLENAME = TABLENAME_PREFIX + "yarn_app_heuristic_result";
+    private static final String HEURISTIC_RESULT_DETAILS_TABLENAME = TABLENAME_PREFIX + "yarn_app_heuristic_result_details";
+
+    private static final String SELECT_HEURISTIC_RESULT_SQL = "SELECT * FROM " + HEURISTIC_RESULT_TABLENAME
+            + " WHERE yarn_app_result_id = :yarn_app_result_id";
+    private static final String DELETE_HEURISTIC_RESULT_SQL = "DELETE FROM " + HEURISTIC_RESULT_TABLENAME
+            + " WHERE yarn_app_result_id = :yarn_app_result_id";
+    private static final String SELECT_HEURISTIC_RESULT_DETAILS_SQL = "SELECT * FROM " + HEURISTIC_RESULT_DETAILS_TABLENAME
+            + " WHERE yarn_app_heuristic_result_id = :yarn_app_heuristic_result_id";
+    private static final String DELETE_HEURISTIC_RESULT_DETAILS_SQL = "DELETE FROM " + HEURISTIC_RESULT_DETAILS_TABLENAME
+            + " WHERE yarn_app_heuristic_result_id = :yarn_app_heuristic_result_id";
+
+    public GarmadonTransferHeuristic() {
+    }
+
+    public void transfert() {
```

Comment:
Since there is no fields, transfer method could be static, this way no need to instanciate GarmadonTransfer object